en/micro-integrator/docs/references/connectors/file-connector/file-connector-example.md

```diff
@@ -26,10 +28,56 @@ Follow these steps to set up the ESB Solution Project and the Connector Exporter
-    <img src="../../../../assets/img/connectors/adding-an-api.png" title="Adding a Rest API" width="800" alt="Adding a Rest API"/>
+## Creating the Integration Logic
+
+1. Right click on the created Integration Project and select, -> **New** -> **Rest API** to create the REST API. 
+    <img src="/assets/img/connectors/adding-an-api.png" title="Adding a Rest API" width="800" alt="Adding a Rest API"/>
+
+2. Provide the API name as File Connector and the API context as `/fileconnector`.
+
+3. First we will create the `/create` resource. Right click on the API Resource and go to **Properties** view. We use a URL template called `/create` as we have two API resources inside single API. The method will be `Post`. 
+    <img src="/assets/img/connectors/filecon-3.png" title="Adding the API resource." width="800" alt="Adding the API resource."/>
+
+4. In this operation we are going to receive input from user which are `source` and `inputContent`. 
+    - source - location of the file is going to be created
+    - inputContent - what needs to be written to the file. 
+
+5. The above two parameters are saved to properties. Therefore drag and drop property mediator to the design view and do as shown below. For further reference, you can read about [Property mediator](https://ei.docs.wso2.com/en/latest/micro-integrator/references/mediators/property-Mediator/).
+    <img src="/assets/img/connectors/filecon-1.png" title="Adding a property" width="800" alt="Adding a property"/>
+
+6. Add the another Property Mediator to get the InputContent value copied. Do the same as in the above step. 
+    - property name: InputContent
+    - Value Type: EXPRESSION
+    - Value Expression: json-eval($.inputContent)
+
+7. Now drag and drop create operation of the File Connector to the Design View as shown below. Set the parameter values as below. We use the property values that we add in step 4 and 5 in this step as `$ctx:source` and `$ctx:inputContent`.
+    <img src="/assets/img/connectors/file-con2.png" title="Adding createFile operation" width="800" alt="Adding createFile operation"/>
+
+8. Finally add a Respond Mediator as user needs to see the response. Now we are done with creating the first API resource, and it is displayed as below. 
+    <img src="/assets/img/connectors/filecon9.png" title="First API Resource" width="800" alt="First API Resource"/>
+
+9. Now we can create the next API resource which is `/read`. From this we are going to read the file content from a user specified location. 
+
+10. As described in step 3, drag and drop another API resource to the design view. Use the URL template as `/read`. The method will be POST. 
+    <img src="/assets/img/connectors/apiResource.png" title="Adding an API resource" width="800" alt="Adding an API resource"/>
+
+11. In this operation user sends the file location as the request payload. It will be written to property as we did in step 10. 
```

Comment:
```suggestion
11. In this operation, the user sends the file location as the request payload. It will be written to the property as we did in step 10. 
```

Reply:
Done